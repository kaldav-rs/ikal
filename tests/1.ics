BEGIN:VCALENDAR
PRODID:-//Nextcloud calendar v1.5.0
VERSION:2.0
CALSCALE:GREGORIAN
BEGIN:VEVENT
CREATED:20170209T192358
DTSTAMP:20170209T192358
LAST-MODIFIED:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
SUMMARY:Vers
CLASS:PUBLIC
STATUS:CONFIRMED
DTSTART;VALUE=DATE-TIME;TZID=Europe/Paris:20150219T190000
DTEND;VALUE=DATE:20170210
END:VEVENT
END:VCALENDAR
