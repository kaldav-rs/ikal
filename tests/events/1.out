Ok(
    VEvent {
        dtstamp: Naive(
            2017-02-09T19:23:58,
        ),
        uid: "5UILHLI7RI6K2IDRAQX7O",
        dtstart: Date(
            2017-02-09,
        ),
        class: Some(
            Public,
        ),
        created: Some(
            Naive(
                2017-02-09T19:23:58,
            ),
        ),
        description: None,
        geo: None,
        last_modified: Some(
            Naive(
                2017-02-09T19:23:58,
            ),
        ),
        location: None,
        organizer: None,
        priority: None,
        sequence: None,
        status: Some(
            Confirmed,
        ),
        summary: Some(
            "Vers",
        ),
        transp: None,
        url: None,
        recurid: None,
        rrule: None,
        dtend: Some(
            Date(
                2017-02-10,
            ),
        ),
        duration: None,
        attach: [],
        attendee: [],
        categories: [],
        comment: [],
        contact: [],
        exdate: [],
        rstatus: [],
        related_to: [],
        resources: [],
        rdate: [],
        x_prop: {},
        iana_prop: {},
    },
)
